signals/content/subject_contains_org_display_name.yml

```diff
@@ -0,0 +1,4 @@
+name: "Subject: Matches entry in the $org_display_names list"
+type: "query"
+source: |  
+  any($org_display_names, strings.icontains(subject.subject, . ))
```

Comment:
This list is probably too large for this to be performant for `any` checks like this

Reply:
I would say stick with `in`, `in~` with large lists for now

Reply:
Awesome! Yeah, I wasn't sure the best way to do that